PGQuestionGenerator.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
+const faker = require('faker')
+const argv = require('yargs').argv
+
+const lines = argv.lines || 10
+const filename = argv.output || 'PGQuestions.csv'
+const stream = fs.createWriteStream(filename)
+const imageFilePath = 'https://ghrsea13-sdc-group2.s3-us-west-2.amazonaws.com/';
+var j = 999;
+
+const createPost = () => {
+    const tripId = faker.random.number(100)
+    const username = faker.internet.userName()
+    const profilePic = imageFilePath + j + '.jpg'
+    const date = faker.date.past().toDateString()
+    const location = faker.address.city()
+    const numContributions =faker.random.number(10)
+    const numHelpfulVotes =faker.random.number(10)
+    const question = faker.lorem.paragraph()
+    j--
+    if (j < 0) {
+        j = 999
+    };
+    return `${tripId},${username},${profilePic},${date},${location},${numContributions},${numHelpfulVotes},${question}\n`
+
+}
+
+const startWriting = (writeStream, encoding, done) => {
```

Comment:
since you use this same function a few times, I suppose you could just write it once and then export it out/import it into other relevant files, not necessary, just pointing out!